DSL/DataMapper/js/validation/validateStoriesRules.js

```diff
@@ -2,22 +2,21 @@ import express from 'express';
 
 const router = express.Router();
 router.post('/', (req, res) => {
-    const steps = req.category === 'rules' ? req.body.rules.steps : req.body.stories.steps;
+    const steps = req.category === 'rules' ? req.body.rule.steps : req.body.story.steps;
     const isValid = validateStepsForNoConsecutiveDuplicates(steps);
     res.json({ result: isValid });
 });
 
 function validateStepsForNoConsecutiveDuplicates(steps) {
-
     for (let i = 1; i < steps.length; i++) {
         const currentStep = steps[i];
         const previousStep = steps[i - 1];
 
         if (currentStep.entities && previousStep.entities && areConsecutive(currentStep, previousStep)) {
-            const currentEntities = currentStep.entities.map(entity => entity.entity);
-            const previousEntities = previousStep.entities.map(entity => entity.entity);
+            const currententities = currentstep.entities.map(entity => entity.entity);
+            const previousentities = previousstep.entities.map(entity => entity.entity);
 
-            if (hasCommonElement(currentEntities, previousEntities)) {
+            if (hascommonelement(currententities, previousentities)) {
```

Comment:
I think naming should remain same in terms of readability

Reply:
fixed it in the next PR